docker-compose.services.yml

```diff
@@ -74,9 +74,23 @@ services:
       - 'matomo_data:/bitnami/matomo'
     depends_on:
       - mariadb
+# CMS-specific services      
+  keystone-db:
+    container_name: keystone-db
+    image: postgres:latest
```

Comment:
Can we please pin this rather than `latest`?

Reply:
my b - thanks for catching! is pinning to `postgres:14` sufficient?

Reply:
Yes. It may be that we find ourselves here with a newer version but that wouldn't necessarily be a bad thing. PostgreSQL is often very good with forward/backward compatibility and I don't think we're doing anything too exotic.